ci/vars-zuul-common.yml

```diff
@@ -9,3 +9,5 @@ sgo_dir: "{{ ansible_env.HOME }}/{{ zuul.projects['github.com/infrawatch/smart-g
 sg_core_dir: "{{ ansible_env.HOME }}/{{ zuul.projects['github.com/infrawatch/sg-core'].src_dir }}"
 sg_bridge_dir: "{{ ansible_env.HOME }}/{{ zuul.projects['github.com/infrawatch/sg-bridge'].src_dir }}"
 prometheus_webhook_snmp_dir: "{{ ansible_env.HOME }}/{{ zuul.projects['github.com/infrawatch/prometheus-webhook-snmp'].src_dir }}"
+__service_telemetry_storage_persistent_storage_class: "crc-csi-hostpath-provisioner"
```

Comment:
Are there changes anywhere else that make this work, or is setting the correct storage class all that was really required? I saw there was previously a define_pvs.yml, but I guess it turned out not to be necessary?

Reply:
No it will not fail, we don't have any pv in crc but we have this storageclass which support dynamic provision so the pvs is being created once the alertmanager or prometheus request a storage.

Reply:
I think Emma was suggesting that if the PVs didn't dynamicly auto-provision, our tests would fail, and I agree. Anwyays, it's great news that this works and there is so little to maintain!

Reply:
Apologies for any misunderstanding. I was providing Chris some background info and then providing justification for this working. 

This looks deceivingly simple, but, if it didn't do what we expected (i.e. create the PVs), then we'd know because the job would fail.
The job passing proves that this is doing exactly what is expected.

Thanks for your research, implementation and testing, Marihan